test/e2e/apimachinery/resource_quota.go

```diff
@@ -53,7 +53,12 @@ var extendedResourceName = "example.com/dongle"
 var _ = SIGDescribe("ResourceQuota", func() {
 	f := framework.NewDefaultFramework("resourcequota")
 
-	ginkgo.It("should create a ResourceQuota and ensure its status is promptly calculated.", func() {
+	/*
+		Release: v1.16
+		Testname: ResourceQuota, object count quota, resourcequotas
+		Description: Create a ResourceQuota. Creation MUST be successful and it's ResourceQuotaStatus MUST match to expected used and total allowed resource's quota count within namespace.
```

Comment:
s/it's ResourceQuotaStatus/its ResourceQuotaStatus/ and similarly for the descriptions below?

Reply:
Done. it's minor but most valuable suggestion. Thanks.